src/bucket/test/BucketManagerTests.cpp

```diff
@@ -508,86 +508,6 @@ TEST_CASE("bucketmanager reattach to running merge", "[bucket][bucketmanager]")
     });
 }
 
-TEST_CASE("bucketmanager reattach HAS from publish queue to finished merge",
```

Comment:
Unclear why this test is being entirely eliminated; shouldn't it still be a thing we want to test when running in a pre-protocol-12 version?

Reply:
fixed